src/Nowruz/modules/orgSettings/components/team/useTeam.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+import { useSelector } from 'react-redux';
+import {
+    addOrganizationMember,
+    CurrentIdentity,
+    filterFollowings,
+    getOrganizationMembers,
+    MembersRes,
+    FollowingRes,
+    removeOrganizationMember,
+} from 'src/core/api';
+import { RootState } from 'src/store';
+
+import { convertFollowingsToContactList } from './memberList';
+
+export const useTeam = () => {
+
+    const currentIdentity = useSelector<RootState, CurrentIdentity | undefined>((state) => {
+        return state.identity.entities.find((identity) => identity.current);
+    });
+    const [members, setMembers] = useState<MembersRes>();
+    const [followings, setFollowings] = useState<FollowingRes>();
+    const [memberList, setMemberList] = useState();
+    const [role, setRole] = useState([{ value: 'Admin' }]);
+    const [userId, setUserId] = useState('');
+
+    const orgId = currentIdentity?.id;
+
+    const getMembers = async () => {
+        console.log('orgId', orgId);
```

Comment:
need this `console.log()`?

Reply:
line-39, line-47 too

Reply:
Thanks for mentioning this, in fact we don't need this component at all, created by Mozhgan before, I removed the whole component and its related codes in other pages.